instrumentation/netty/netty-3.8/src/main/java/io/opentelemetry/auto/instrumentation/netty/v3_8/ChannelFutureListenerInstrumentation.java

```diff
@@ -66,7 +66,7 @@ public ChannelFutureListenerInstrumentation() {
       packageName + ".AbstractNettyAdvice",
       packageName + ".ChannelTraceContext",
       packageName + ".ChannelTraceContext$Factory",
-      packageName + ".client.NettyHttpClientDecorator",
+      packageName + ".client.NettyHttpClientTracer",
```

Comment:
this should fix the muzzle failure, i will go ahead and commit this suggestion to validate
```suggestion
      packageName + ".client.NettyHttpClientTracer",
      packageName + ".client.NettyResponseInjectAdapter",
```